app/controllers/nonsipp/bonds/BondsListController.scala

```diff
@@ -76,37 +77,43 @@ class BondsListController @Inject()(
     current: Int,
     previous: Int
   ): Action[AnyContent] = identifyAndRequireData(srn, mode, year, current, previous) { implicit request =>
-    onPageLoadCommon(srn, page, mode)(implicitly)
+    val viewOnlyViewModel = ViewOnlyViewModel(
+      viewOnlyUpdated = request.previousUserAnswers match {
+        case Some(previousUserAnswers) =>
+          getCompletedOrUpdatedTaskListStatus(
+            request.userAnswers,
+            previousUserAnswers,
+            pages.nonsipp.bonds.Paths.bondTransactions
```

Comment:
I think this needs an additional parameter to exclude bond disposals. (was an existing bug)
E.g.  https://github.com/hmrc/pension-scheme-return-frontend/blob/c0dfddbea056b61af444f3276f50053d373ccc0f/app/utils/nonsipp/TaskListStatusUtils.scala#L1092